README.md

```diff
@@ -60,18 +60,36 @@ cannot open 'zroot/iocell/jails/XXXXXXXX-XXXX-XXXX-a5be-3417ebb13845': dataset d
 
 **QUICK HOWTO:**
 
-Fetch a release:
+`iocell(8)` requires a zfs pool to store its jails, templates and configuration.
+FreeBSD's default installer uses `zroot` as the default name for the system
```

Comment:
Change to "FreeBSD's installer uses `zroot` as the default name" as there is are other official installers for FreeBSD.